dask_sql/context.py

```diff
@@ -800,8 +800,19 @@ def _get_ral(self, sql):
         # get the schema of what we currently have registered
         schemas = self._prepare_schemas()
 
-        # Now create a relational algebra from that
-        generator_builder = RelationalAlgebraGeneratorBuilder(self.schema_name)
+        # True if the SQL query should be case sensitive and False otherwise
+        case_sensitive = self.schema[self.schema_name].config.get_config_by_prefix(
+            "dask.sql.identifier.case.sensitive"
+        )
+        case_sensitive = (
+            False
+            if case_sensitive == {}
+            else case_sensitive["dask.sql.identifier.case.sensitive"]
+        )
```

Comment:
Minor simplification:

```suggestion
        case_sensitive = self.schema[self.schema_name].config.get_config_by_prefix(
            "dask.sql.identifier.case.sensitive"
        ).get("dask.sql.identifier.case.sensitive", False)
```

Reply:
Also note that depending on the answer to https://github.com/dask-contrib/dask-sql/pull/316#discussion_r764347926 we might want to change the default in the `get` to True.